app/_components/Home/Sponsor/index.jsx

```diff
@@ -1,9 +1,29 @@
-// TODO: Create a Sponsor component.
-export default function Sponsor() {
-    return (
-      <main>
-        <h1 id="sponsor" className="text-6xl font-bold">Sponsor</h1>
-      </main>
-    );
-  }
-  
\ No newline at end of file
+import Image from "next/image";
+import sponsorData from "../../../data/home/sponsors.json";
+//Use Transparent sponsor logos for best effect
+const Sponsors = () => {
+  return (
+    <div className="text-center justify-center py-8  ">
+      {Object.keys(sponsorData).map((category) => (
+        <div key={category} className="">
+          <h2 className="text-6xl font-bold">{category}</h2>
+          <div className="flex flex-row justify-center py-8 bg-sponsors-bg bg-cover bg-center overflow-hidden">
+            {sponsorData[category].map((Sponsor) => (
+              <div key={Sponsor.id} className="items-center mx-4">
+                <Image
+                  className="rounded-xl"
+                  src={Sponsor.src}
```

Comment:
Define a default value for `sponsor.src` since `sponsors.json` is a dynamic and changeable file. 
Example:
`src={sponsor.src || '/default-image.jpg'}`